bin/packages/post-css-config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = [
+	require( './node_modules/@wordpress/postcss-themes' )( {
+		defaults: {
+			primary: '#0085ba',
+			secondary: '#11a0d2',
+			toggle: '#11a0d2',
+			button: '#0085ba',
+			outlines: '#007cba',
+		},
+		themes: {
+			'woocommerce-page': {
```

Comment:
So this means we could potentially add another theme to target the calypsoify env to make things match colors there?

Reply:
I think so – I think it's based on a body class. I'm not entirely sure how this part works, I copied this from @psealock's work on the postcss theme, but since nothing here relies on CSS (yet) I haven't been able to test it.

Reply:
>So this means we could potentially add another theme to target the calypsoify

Yes, exactly.

>but since nothing here relies on CSS (yet) I haven't been able to test it.

The GB buttons are themed using this post-css function and config. I tested this out by adding a new theme 

```js
'testing-add-lerna': {
    primary: '#4ade1d',
    secondary: '#4ade1d',
    toggle: '#4ade1d',
    button: '#4ade1d',
    outlines: '#4ade1d',
},
```
I can see the correctly constructed CSS rules in `/dist/css/index.css` and `/dist/css/embedded.css`:

```
body.testing-add-lerna .components-button.is-primary { ... }
```

For some reason those rules aren't being added to `dist/app/index.css` or `dist/components/index.css` and so I can't see the theme expressed in wc-admin.

Also, the old `postcss.config.js` can be removed.

Reply:
> For some reason those rules aren't being added to dist/app/index.css or dist/components/index.css and so I can't see the theme expressed in wc-admin.

Which file did you add the new theme to?

> Also, the old postcss.config.js can be removed.

Once the components are built from packages, sure – right now they shouldn't be using this build process though, which is what I meant by not being able to test it.

Reply:
Oh I see now, thanks. So if I were to add a new theme now, it would have to be to both postcss configs. In any case, that means the new `bin/packages`/post-css-config.js` is working as expected.